models/doctype/Item/Item.js

```diff
@@ -111,10 +111,17 @@ export default {
         }
       },
     },
+    {
+      fieldname: 'itemCode',
+      label: 'Item Code',
+      fieldtype: 'Int',
+      placeholder: 'Item Code',
+    },
```

Comment:
Any reason for _Int_ `itemCode`? Also any reason for `itemCode`, in ERPN `item_code` is used as a unique identifier, here `name` (_Item Name_) already serves that purpose.

Imo if this is required we can use _Data_ for an alphanumeric identifier.